warehouse/models/gtfs_views/_gtfs_views.yml

```diff
@@ -301,6 +301,17 @@ models:
         description: |
           Days since the feed's contents (the contents of any loadable file
           within the feed) have changed, relative to date
+      - name: is_service_valid
+        description: |
+          Whether the service is valid (active) on this date (boolean). Service date is based on when feed_key is active.
+      - name: days_from_service_start_date
+        description: |
+          Number of days between date and service start date. if is_service_valid is False, this may be negative
```

Comment:
Would specify: service start date is the first day on which service occurs on any service in this feed (just because "service" is also used to mean like an individual service pattern identified by a `service_id`.... may just be worth explicitly stating that this is a feed-wide summary.... That should maybe be obvious given what this table is but these are all kind of confusing so I think it's worth spelling out)